### PR TITLE
[Issue #1751] Turn on feature flag for delivery metrics

### LINKED ISSUE #1751: Turn on feature flag for new merge strategy
### Summary

We want to turn on the feature flag for sprinty mcburndown reports so that we're hiding deliverables that are not in progress or planning.

### Acceptance criteria

- [ ] The daily reports use the new feature flag
- [ ] Deliverable percent complete only shows in progress and planning deliverables

### PR DESCRIPTION
**Pull request by widal001**
## Summary

Turns on feature flag to limit sprint reporting by deliverable status. Here's a [sample output posted to slack](https://betagrantsgov.slack.com/archives/C06L7JGT9M0/p1713361826499229)

Fixes #1751 

### Time to review: __5 mins__

- Adds a `make roadmap-data-export` command to `./analytics/Makefile`
- Adds a `roadmap-data-export` as an additional target for `make gh-data-export`
- Adds feature flags to `make percent-complete` to use new join strategy and limit statuses to "In Progress" and "Planning"

1. Checkout the PR locally
2. Run `make setup` to install the package
3. Set up your `.secrets.toml` file as described in the `development.md` docs
4. Run `make sprint-reports-with-latest-data ACTION=show-results` to display the results with the new join path

Here's what the output should look like in slack:

Screenshot 2024-04-17 at 9 58 58 AM

Here's an example of the updated report: 

Screenshot 2024-04-17 at 9 59 39 AM

### CODE REVIEW

**Comment by widal001** [comment]
@coilysiren I noticed [this failure](https://github.com/HHS/simpler-grants-gov/actions/runs/8723261663/job/23931268938?pr=1764) from the security scans, and based on my understanding of the error it's flagging an issue with `idna` (part of the python standard library) on version 3.6 

I thought that we were using python 3.12 for our Docker image. Do we need to pin the python version somewhere else in these scans to avoid raising this flag or am I misunderstanding the failure?

Screenshot 2024-04-17 at 10 56 47 AM

**Comment by coilysiren** [comment]
@widal001 it's flagging the version of idna, not the version of python. The fix is to update idna. You can use `poetry` to do that.

**Review by coilysiren — DISMISSED** [review]
I can approve again when CI is passing

File: analytics/Makefile [hunk 1]
```diff
@@ -5,7 +5,9 @@
 ORG ?= HHS
 REPO ?= simpler-grants-gov
 SPRINT_PROJECT ?= 13
+ROADMAP_PROJECT ?= 12
 SPRINT_FILE ?= data/sprint-data.json
+ROADMAP_FILE ?= data/roadmap-data.json
```
Comment (coilysiren) [inline analytics/Makefile:10]: The step functions can't write to the `data/` directory. You need to change the directory for `ROADMAP_FILE` from `data/roadmap-data.json` in this file: https://github.com/HHS/simpler-grants-gov/blob/main/infra/analytics/service/sfn_sprint_reports.tf#L36

Or, even better, making it resolve to `${OUTPUT_DIR}/roadmap-data.json`, so you don't need to add a new makefile variable into the terraform file at all.

Comment (widal001) [reply analytics/Makefile:10]: Great thanks for flagging this! I'll update

Comment (widal001) [reply analytics/Makefile:10]: Should I update that for the other files too?

Comment (coilysiren) [reply analytics/Makefile:10]: If you care about consistency, yes.

Comment (widal001) [reply analytics/Makefile]: Okay cool, I updated them in this commit: [refactor: Parametrizes output directory in Makefile](https://github.com/HHS/simpler-grants-gov/pull/1764/commits/ba66758efb642b730d5c8264e8b3d58a1a0ff514)

**Review by coilysiren — APPROVED** [review]

**Comment by AlexanderStephensonUSDS** [comment]
LGTM. Documentation also looks clear.

**Pull request merged**